docs/ember/component-signatures.md

```diff
@@ -67,6 +67,32 @@ export default class SuperTable<T> extends Component<SuperTableSignature<T>> {}
 
 {% endcode %}
 
+{% code title="app/components/simple-hello.ts" %}
+
+```typescript
+import Component from '@glimmer/component';
+
+export interface SimpleHelloSignature {
+  // We have a `<div>` as our root element
+  Element: HTMLDivElement;
+  // We accept no arguments
+  Args: Record<string, never>;
```

Comment:
I think I would omit `Args` here as well. Including it but not `Blocks` seems confusing as the two have pretty much identical usages (aside from their contents of course), and as far as I know you never need to do this -- the scenario we discussed over discord is where you also override the constructor (not the most common/simplest case), in which case you can just do

```ts
interface SimpleHelloSignature {
  Element: HTMLDivElement;
}

export default class SimpleHello extends Component<SimpleHelloSignature> {
  constructor(owner: Owner, args: Record<string, never>) {
    super(owner, args);
  }
}
```

Reply:
Seems fine to me